src/config/index.ts

```diff
@@ -166,6 +166,35 @@ export const RESPONSE_DUE_DATE_FIELD_ID =
  */
 export const GH_USER_TOKEN = process.env.GH_USER_TOKEN || '';
 
+/**
+ * App auth strategy options. For branding purposes, we have one app per org
+ * (getsantry for getsentry, covecod for codecov, &c). Down in getClient we
+ * will instantiate a GitHub octokit client for each org the first time we need
+ * it, then cache it for the duration of our server process as it seems to be
+ * smart enough to renew auth tokens as needed. When we do that, we will set
+ * installationId. Here in config we need to populate the appId and privateKey
+ * from the environment.
+ *
+ * TODO: Expand from a single app/org to multiple. First we need to clean up
+ * getClient calls to use a dynamic owner/org instead of OWNER as defined above.
+ */
+
+export interface AppAuthStrategyOptions {
+  // I didn't find something great in @octokit/types.
+  appId: number;
+  privateKey: string;
+  installationId?: number;
+}
+
+export const GH_APP_AUTH_OPTIONS = new Map<string, AppAuthStrategyOptions>();
+
+if (process.env.GH_APP_IDENTIFIER && process.env.GH_APP_SECRET_KEY) {
```

Comment:
What happens if one or both of these aren't set? Should we bubble some kind of error up, or is this situation dealt with later when we do operations that require AUTH? If we assume that these must always be set for the GH integration to function properly, may be clearer to put an `else` here and error closer to the source of the problem.

Reply:
This change preserves the status quo, which is that GitHub auth errors aren't discovered until we try to use the client (inside of `getClient`, when we use an unbound client to find an installation ID). I figure this is probably fine since Dev Infra or someone else may want to be able to run `eng-pipes` without configuring GitHub.